docs/usage.rst

```diff
@@ -78,6 +78,25 @@ Optional arguments:
     ``-v, --version``
         Show program's version number and exit
 
+Presenter Console
+-----------------
+
+The presenter console feature is designed for showing an annotated version of
+the presentation on the local laptop display, while showing the presentation
+itself on a projector.
+
+To use this feature, open the presentation in a browser, and press "P". A new
```

Comment:
I might be a bit picky but do you mind changing the "P" for 
```text
`p`
```

Reply:
Done.